src/CMakeLists.txt

```diff
@@ -865,7 +918,7 @@ if(NOT ALLOCATOR STREQUAL "jemalloc")
 endif()
 add_custom_target(build_rocksdb
     COMMAND
-    PORTABLE=1 ${disable_jemalloc} $(MAKE) static_lib EXTRA_CXXFLAGS='-fPIC -Wno-unused-variable'
+    PORTABLE=1 ${disable_jemalloc} $(MAKE) static_lib V=1 VERBOSE=1 CXX=$(CXX) EXTRA_CXXFLAGS='-Irocksdb/include  -I/usr/local/include -fPIC -Wno-unused-variable'
```

Comment:
why you'd need `-Irocksdb/include` and `-I/usr/local/include`?

and could you explain why we need `VERBOSE=1` and `V=1`?